client/blocks/credit-card-form/index.jsx

```diff
@@ -199,15 +203,15 @@ const CreditCardForm = createReactClass( {
 
 				wpcom.updateCreditCard( apiParams, ( apiError, response ) => {
 					if ( apiError ) {
-						if ( this._mounted ) {
+						if ( this.formElem ) {
 							this.setState( { formSubmitting: false } );
 						}
 						notices.error( apiError.message );
 						return;
 					}
 
 					if ( response.error ) {
-						if ( this._mounted ) {
+						if ( this.formElem ) {
```

Comment:
looks like there are lots of these places where an inline `&&` could clear up the noise (in my opinion)

```js
.catch( ( { message } ) => this.formElem && this.setState( { formSubmitting: false } ) )

if ( apiError ) {
	this.formElem && this.setState( { formSubmitting: false } )
	notices.error( apiError.message )
	return;
}

if ( response.error ) {
	this.formElem && this.setState( { formSubmitting: false } )
	notices.error( response.error )
}
```

further if we're doing it so often with `formSubmitting` maybe it makes sense to extract it into its own method

```js
endSubmitting = () => this.formElem && this.setState( { formSubmitting: false } )

.catch( ( { message } ) => this.endSubmitting() … )

if ( apiError ) {
	this.endSubmitting()
	notices.error( … )
	return;
}
```